src/main/java/core/mate/academy/producers/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.producers;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer cat = new Bulldozer();
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(cat);
```

Comment:
Add more than one bulldozer to the list.
Also fix it in ExcavatorProducer and TruckProducer